### PR TITLE
[RAPTOR-8685] refactor test scripts - drop 1

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
DRUM wheel is stashed /unstashed in groovy scripts, so resolve path to DRUM wheel on the groovy script level and pass it as a param into a test bash script.

`if/else` statement in the `test_drop_in_envs` is for compatibility with the jenkins job that uses this script in DTS. This `if/else` will be removed once job is updated to the current logic.

### CODE REVIEW

**Comment by yakov-g** [comment]
jenkins test this

**Comment by yakov-g** [comment]
jenkins run test_drop_in_envs test_inference_model_templates test_integration_general test_integration_python_sklearn test_integration_r_environment test_integration_java_environment test_training_model_templates

**Comment by yakov-g** [comment]
jenkins run test_integration_java_environment

**Comment by yakov-g** [comment]
jenkins run test_integration_java_environment

**Comment by yakov-g** [comment]
jenkins run test_integration_general

**Comment by yakov-g** [comment]
jenkins run test_integration_python_sklearn

**Comment by yakov-g** [comment]
jenkins run test_integration_python_sklearn

**Comment by yakov-g** [comment]
jenkins run test_integration_python_sklearn

**Comment by yakov-g** [comment]
jenkins run test_integration_python_sklearn

**Comment by yakov-g** [comment]
jenkins run test_integration_python_sklearn

**Review by rvorobii — APPROVED** [review]

File: jenkins/test_integration_per_framework.sh [hunk 1]
```diff
@@ -7,39 +7,41 @@
 # Released under the terms of DataRobot Tool and Utility Agreement.
 
 set -ex
-
 GIT_ROOT=$(git rev-parse --show-toplevel)
 
+if [ -z $1 ]; then
```
Comment (zohar-mizrahi) [inline jenkins/test_integration_per_framework.sh:12]: I'd set `$1` and `$2` to variables thus the script will be more readable.

**Comment by elatt** [comment]
Closing due to inactivity.

**Pull request closed without merging**